src/styles/variables.scss

```diff
@@ -125,7 +125,7 @@ $highlight-color: rgba(255, 255, 60, 0.3);
 $highlight-color-second: rgba(206, 206, 60, 0.4);
 $highlight-color-third: rgba(192, 192, 49, 0.4);
 $highlight-color-focus: rgba(156, 230, 255, 0.5);
-$top-bar-height: 40px;
+$top-bar-height: 48px;
```

Comment:
It doesn't look to me like this change is an essential part of this PR. Since it also isn't feature-flagged, I would be inclined to leave this change out of this PR and propose it separately.

Reply:
To answer your question, yes I believe it was done to make more clickable space but I'm not 100% sure as I can't fine an exact reason.

Reply:
I do think it emphasizes the heading of the sidebar and it looks a lot better with it but I'm happy to move it to another PR.
w/o it
![image](https://user-images.githubusercontent.com/30059933/54381502-33610a80-464b-11e9-9f27-50029f03d6b2.png)
w/ it
![image](https://user-images.githubusercontent.com/30059933/54382675-b4210600-464d-11e9-8c91-3e32e3e7f746.png)